racktables/racktables.py

```diff
@@ -14,20 +14,21 @@ def __init__(self, url, username, password):
         self.url = url
 
     def search(self, search_payload={}):
-        params="&".join("%s=%s" % (k, v) for k, v in search_payload.items())
-        req = self.s.get(
-            join(self.url, "index.php"),
-            params=params
-        )
+        params = "&".join("%s=%s" % (k, v) for k, v in search_payload.items())
+        req = self.s.get(join(self.url, "index.php"), params=params)
         status = req.status_code
         if status == 401:
-          raise Exception("Racktables returned 401 Unauthorized. Are your credentials correct?")
+            raise Exception(
+                "Racktables returned 401 Unauthorized. Are your credentials correct?"
+            )
         elif status >= 300:
-          raise Exception(f"Racktables returned statuscode {status} while trying to access {req.request.url}. Manual investigation needed.")
+            raise Exception(
+                f"Racktables returned statuscode {status} while trying to access {req.request.url}. Manual investigation needed."
```

Comment:
just for clarity. it meant to be _statuscode_ or _status code_?

Reply:
https://github.com/os-autoinst/scripts/pull/395#discussion_r2099522002